src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -165,11 +164,15 @@ export default class DiscussionComment extends HTMLElement {
       return;
     }
 
-    const typePill = document.createElement('span');
-    typePill.innerText = COMMENT_TYPES[type].name;
-    typePill.classList.add(...COMMENT_TYPES[type].badgeStyles);
-    typePill.slot = DiscussionComment.#SLOT_TYPE_TAG;
-    this.appendChild(typePill);
+    if (this.typePill) {
+      this.typePill.remove();
+    }
+
+    this.typePill = document.createElement('span');
```

Comment:
It is a bit weird for the method to be typeTag but the variable to be typePill. I would make them have the same name.

Reply:
changed to typeTag